lib/hotdog/commands/tag.rb

```diff
@@ -33,14 +33,16 @@ def run(args=[], options={})
             @db.transaction do
               create_tags(@db, options[:tags])
               options[:tags].each do |tag|
-                associae_tag_hosts(@db, tag, hosts)
+                associate_tag_hosts(@db, tag, hosts)
               end
             end
           end
         end
         hosts.each do |host|
           if options[:tags].empty?
-            # nop
+            # nop; just show current tags
+            host_tags = with_retry { host_tags(host, source=options[:tag_source]) }
+            p host_tags['tags']
```

Comment:
As commented above "nop; just show current tags", it is expected. But it should be puts or something to explicitly output...